resources/dicts/thoughts/alive/general.json

```
@@ -51,87 +49,1424 @@
+            "compassionate"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "competitive_gen",
+        "thoughts": [
+            "Is pestering other cats to challenges around camp",
+            "Challenged a clanmate to a staring contest",
+            "Challenged a clanmate to a staring contest and won",
+            "Challenged a clanmate to a staring contest and lost",
+            "Has been a good sport about losing to a clanmate",
+            "Has been a bit of a bad sport about losing to a clanmate",
+            "Challenged a clanmate to a race",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt and won",
+            "Challenged {PRONOUN/m_c/poss} clanmates to a scavenger hunt and lost",
+            "Makes every patrol into a competition",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Puffs out {PRONOUN/m_c/poss} chest, feeling secure with the space {PRONOUN/m_c/subject} {VERB/m_c/take/takes} up",
+            "Watched a clanmate do a chore they're really good at to see how {PRONOUN/m_c/object} can improve",
+            "Is practicing doing things around camp as fast as possible just in case"
+        ],
+        "main_trait_constraint": [
+            "competitive"
+        ]
+    },
+    {
+        "id": "confident_warrior",
+        "thoughts": [
+            "Is puffing {PRONOUN/m_c/poss} chest out",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Thinks that {PRONOUN/m_c/subject} {VERB/m_c/are/is} the smartest cat in the Clan",
+            "Thinks that {PRONOUN/m_c/subject} {VERB/m_c/are/is} the funniest cat around",
+            "Is strutting around confidently",
+            "Thinks that {PRONOUN/m_c/subject} {VERB/m_c/are/is} the fastest runner in the Clan",
+            "Puffs out {PRONOUN/m_c/poss} chest, feeling secure with the space {PRONOUN/m_c/subject} {VERB/m_c/take/takes} up",
+            "Doesn't falter when challenged on {PRONOUN/m_c/poss} abilities",
+            "Offers tips to every cat in the Clan"
+        ],
+        "main_trait_constraint": [
+            "confident"
+        ]
+    },
+    {
+        "id": "cunning_gen",
+        "thoughts": [
+            "Convinced a clanmate to get {PRONOUN/m_c/object} something {PRONOUN/m_c/subject} could easily have gotten {PRONOUN/m_c/self}",
+            "Is always five steps ahead of {PRONOUN/m_c/poss} clanmates",
+            "Knows exactly how to get {PRONOUN/m_c/self} of a mess {PRONOUN/m_c/subject} orchestrated",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the clan",
+            "Got a clanmate to do {PRONOUN/m_c/poss} work with a false promise {PRONOUN/m_c/subject} never intended to fufill, knowing {PRONOUN/m_c/subject} could talk {PRONOUN/m_c/self} out of it easy"
```

```suggestion
            "Got a Clanmate to do {PRONOUN/m_c/poss} work with a false promise {PRONOUN/m_c/subject} never intended to fulfill, knowing {PRONOUN/m_c/subject} could talk {PRONOUN/m_c/self} out of it easy"
```
this one is a little long?

---

definitely true, will shorten it in a sec